components/myguiplatform/myguirendermanager.cpp

```diff
@@ -128,8 +128,11 @@ class Drawable : public osg::Drawable {
             osg::Texture2D* texture = batch.mTexture;
             if(texture)
                 state->applyTextureAttribute(0, texture);
-
+#if defined (ANDROID) || defined (OPENGL_ES)
+            osg::GLBufferObject* bufferobject = 0;
```

Comment:
it is bug in osg in gles render , for example mali t760 supports vbo , but  game crashes inside osg sources